README.md

```diff
@@ -19,9 +19,6 @@ git clone https://github.com/launchdarkly/hello-phoenix.git
 
 Install dependencies
 ```
-cd hello-phoenix/assets
```

Comment:
The new Phoenix template project did not require any NPM stuff. Part of why this is now nearly 15k lines smaller.